src/software/ai/navigator/path_planner/theta_star_path_planner_test.cpp

```diff
@@ -197,14 +224,20 @@ TEST_F(TestThetaStarPathPlanner,
     Field field = Field::createSSLDivisionBField();
     Point start{0, 0}, dest{3, 0};
 
-    // Place a rectangle over our destination location
-    std::vector<ObstaclePtr> obstacles = {
-        robot_navigation_obstacle_factory.createFromShape(
-            Rectangle(Point(1, -1), Point(2, 1)))};
+    Polygon obstacle_shape = Rectangle(Point(1, -1), Point(2, 1));
+
+    std::vector<ObstaclePtr> smaller_obstacles = {
+        robot_navigation_obstacle_factory.createFromShape(obstacle_shape)};
```

Comment:
please add some comments on the larger and smaller obstacle system

Reply:
Added clarification comments and changed implementation slightly for the tests.